Content.Shared/Interaction/Events/GettingAttackedAttemptEvent.cs

```diff
@@ -0,0 +1,10 @@
+namespace Content.Shared.Interaction.Events;
+
+/// <summary>
+/// Raised directed on the target entity when being attacked.
+/// </summary>
+[ByRefEvent]
+public record struct GettingAttackedAttemptEvent(bool Cancelled)
+{
+    public bool Cancelled = Cancelled;
+}
```

Comment:
You shouldn't need to write out the field here.

```suggestion
public record struct GettingAttackedAttemptEvent(bool Cancelled);
```